src/workerd/io/compatibility-date-test.c++

```diff
@@ -191,7 +191,8 @@ KJ_TEST("encode to flag list for FL") {
 
   auto compileOwnFeatureFlags = [&](kj::StringPtr compatDate,
       kj::ArrayPtr<const kj::StringPtr> featureFlags,
-      CompatibilityDateValidation dateValidation = CompatibilityDateValidation::FUTURE_FOR_TEST) {
+      CompatibilityDateValidation dateValidation = CompatibilityDateValidation::FUTURE_FOR_TEST,
+      bool experimental = true) {
```

Comment:
Let's make the default `false` here too.

Reply:
Done